headers/cub3d.h

```diff
@@ -74,13 +83,31 @@ enum e_mm {
 	PLAYER
 };
 
+// data struct to draw into an image
+typedef struct s_img_data {
+	void	*img;
+	char	*addr;
+	int		bits_per_pixel;
+	int		line_length;
+	int		endian;
+}	t_img_data;
+
+// Store all the textures informations:
+// - the path to the texture
+// - the pointer to the image
 typedef struct s_texture {
 	char	*north;
+	void	*img_north;
 	char	*south;
+	void	*img_south;
 	char	*east;
+	void	*img_east;
 	char	*west;
+	void	*img_west;
 	char	*floor;
+	void	*img_floor;
 	char	*ceil;
+	void	*img_ceil;
```

Comment:
Do we do a better structure ?

Reply:
Yes we do !

Reply:
00868b206bc446c663fd618ddf212e01e5071c5a